src/controllers/webviewController.ts

```diff
@@ -28,16 +29,18 @@ export class WebviewPanelController implements vscode.Disposable {
     private _disposables: vscode.Disposable[] = [];
     private _isDisposed: boolean = false;
     private _isActive: boolean;
+    private _rendered: boolean = false;
 
     constructor(
+        private _vscodeWrapper: VscodeWrapper,
         uri: string,
```

Comment:
So if you're using the wrapper I'd suggest updating all the current calls to use the wrapper as well (such as vscode.workspace.getConfiguration on line 41)